src/java/org/apache/cassandra/net/SensorsCustomParams.java

```diff
@@ -139,6 +141,9 @@ public static String encodeTableInInternodeBytesTableParam(String tableName)
      */
     public static <T> void addWriteSensorToResponse(Message.Builder<T> response, RequestSensors sensors, Context context)
     {
+        Preconditions.checkNotNull(sensors);
```

Comment:
because this is a public utility method that anyone can call (not strictly related to this PR, but I found it cleaner this why), and it will NPE if those were null. It makes sense to check response too.